hack/e2e.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+CLUSTER=${CLUSTER:-"e2e-mo-kind"}
+MO_VERSION=${MO_VERSION:-"nightly-20eeb7c9"}
+IMAGE_REPO=${IMAGE_REPO:-"matrixorigin/matrixone"}
+OPNAMESPACE=${OPNAMESPACE:-"e2e-matrixone-operator"}
```

Comment:
I recommend we set the default to `mo-system`, which is also recommended by our helm chart